discourse-doctor.rb

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env ruby
+# coding: utf-8
```

Comment:
The interpreter just looks for `coding:`, but yes, `encoding` is clearer and it shouldn't be necessary in this case.